### PR TITLE
Create test-5.js

### PR DESCRIPTION
**Pull request by daint-1849**

### CODE REVIEW

**Review by daint2git — COMMENTED** [review]
abc

**Review by daint2git — COMMENTED** [review]
123

**Review by daint2git — COMMENTED** [review]
123

**Review by daint-1849 — COMMENTED** [review]
mmm

File: examples/test-5.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Test 5');
```
Comment (daint-1849) [inline examples/test-5.js:1]: 123

Comment (daint2git) [reply examples/test-5.js:1]: adv

**Review by daint2git — CHANGES_REQUESTED** [review]

**Review by daint2git — APPROVED** [review]

**Review by daint2git — COMMENTED** [review]
123

File: examples/test-5.js (hunk 1 above)
Comment (daint2git) [reply examples/test-5.js:1]: 123

Comment (daint2git) [reply examples/test-5.js:1]: 123

**Pull request closed without merging**